urbanaccess/gtfs/utils_format.py

```diff
@@ -406,23 +411,34 @@ def add_unique_agencyid(agency_df=None,stops_df=None,routes_df=None,trips_df=Non
 
         if len(agency_df['agency_name']) == 1:
             assert agency_df['agency_name'].isnull().values == False
+
+            # TODO: Again, this need to be moved into a helper function
```

Comment:
This TODO refers to a specific unique_agency_id generation that relies on the agency column at index 0 which is different from the agency name function you created below. However, this TODO can stay here because it would be good to add a parameter to your function to accommodate a call to a index so that this can be folded into your function.